app/models/aws_config.rb

```diff
@@ -53,4 +69,207 @@ def self.s3_host_name
     end
     s3_host_name
   end
+
+  ## SQS
+  def self.sqs
+    @sqs ||= Aws::SQS::Client.new(region: @config.region)
+  end
+
+  def self.send_message(message)
+    resp = self.sqs.send_message({
+      queue_url: @config.sqs_url,
+      message_body: message,
+    },)
+    resp
+  end
+
+  def self.receive_message(max_number = 10)
+    resp = self.sqs.receive_message(queue_url: @config.sqs_url, visibility_timeout: 600, max_number_of_messages: max_number)
+    resp
+  end
+
+  def self.message_exist?(resp)
+    if !resp || resp.messages.count == 0
+      false
+    else
+      true
+    end
+  end
+
+  def self.delete_message(message_object)
+    resp = self.sqs.delete_message({
+      queue_url: @config.sqs_url,
+      receipt_handle: message_object.receipt_handle,
+    },)
+    resp
+  end
+
+  def self.batch_delete(entries)
+    self.sqs.delete_message_batch(queue_url: @config.sqs_url, entries: entries)
+  end
+
+  ## S3
+  #
+  def self.upload_files(bucket, files, prefix)
+    files.each do |f|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'f' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)